include/sway/tree/container.h

```diff
@@ -72,9 +72,13 @@ struct sway_container {
 	// For C_OUTPUT, this is the output position in layout coordinates
 	// For other types, this is the position in output-local coordinates
 	double x, y;
+	double saved_x, saved_y;
 	// does not include borders or gaps.
 	double width, height;
 
+	// For C_WORKSPACE only
```

Comment:
Then it should be moved to a new struct and stuck in the union at the top of this struct